tools/build.sh

```diff
@@ -86,15 +86,15 @@ _generate() {
         --java_out=$PROTOC_OUT \
         $proto_src
 
-    # echo "# - validation tests"
-    # $PROTOC_PATH \
-    #     --plugin=protoc-gen-java-snowpea=protoc-gen-java-snowpea/startup-script.sh \
-    #     --java-snowpea_out $TEST_GEN_OUT \
-    #     --java-snowpea_opt "MessagePublishedData,LogEntryData" \
-    #     -I $PROTOBUF_SRC \
-    #     -I $THIRDPARTY \
-    #     --experimental_allow_proto3_optional \
-    #     $proto_src
+    echo "# - validation tests"
+    $PROTOC_PATH \
+        --plugin=protoc-gen-java-snowpea=protoc-gen-java-snowpea/startup-script.sh \
+        --java-snowpea_out $TEST_GEN_OUT \
+        --java-snowpea_opt "MessagePublishedData,LogEntryData#strict" \
+        -I $PROTOBUF_SRC \
+        -I $THIRDPARTY \
+        --experimental_allow_proto3_optional \
```

Comment:
What are the terms of this experiment? I assume in the future this flag is likely to change or no longer be needed, but might their be a decision about the experiment to remove the capability we're depending on?

Reply:
Not all plugins support optional fields in proto3. This flags that my plugin does support optionals. The flag will become the default soon.